FusionIIIT/applications/academic_procedures/views.py

```diff
@@ -694,7 +699,6 @@ def branch_change_request(request):
     else:
         messages.info(request, 'Unable to proceed')
```

Comment:
The messages we are sending doesn't make any sense, because we are not rendering them on the `/main` page, in fact, we are not even passing them to the `/main`. This could be an issue itself.

Although for now, you can remove the message line, because we are already sending them through the HTML file.

Reply:
Updated.